util/devel/check_paths

```diff
@@ -109,22 +109,34 @@ def check_token(t):
 
 def special_exception(filename, token):
     """Exception is a synonym of hack"""
-    if filename == "doc/rst/developer/adding-a-tasking-model.txt":
+    if filename == "doc/rst/developer/implementation/adding-a-tasking-model.rst":
         # Allow this file's wonderful paths
         if "wonderful" in token:
             return True
-    elif filename == "doc/rst/developer/adding-a-comm-layer.txt":
+    elif filename == "doc/rst/developer/implementation/adding-a-comm-layer.rst":
         # And this one's speedy paths
         if "speedy" in token:
             return True
+    elif filename == "doc/rst/developer/implementation/adding-a-launcher.rst":
+        # And this one's <name>s
+        if "<name>" in token:
```

Comment:
If we're now splitting on '<' and '>' (in `check_line()`) does this case actually occur?